pkg/awsClients/ssm/ssm.go

```diff
@@ -151,14 +151,11 @@ func Connect(args utils.Args, target utils.Target) {
 		target.SessionInfo = string(session_raw)
 	}
 
-	connect_args := []string{
-		"session-manager-plugin",
+	session.ValidateInputAndStartSession(
 		target.SessionInfo,
-		config.Region,
```

Comment:
Just checking. Intentionally removing region?

Reply:
its now handled by the initializing of the sdk rather than being passed in explicitly to the plugin